integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -63,4 +72,65 @@ public void remove_key(String key) throws IOException {
+    public void user_sends_a_valid_request_to_authorization_end_point()
+            throws IOException, InterruptedException, URISyntaxException {
+        List<NameValuePair> params = new ArrayList<>();
+        params.add(
+                new BasicNameValuePair(
+                        "redirect_uri",
+                        "https://di-ipv-core-stub.london.cloudapps.digital/callback"));
+        params.add(new BasicNameValuePair("client_id", "ipv-core-stub"));
+        params.add(new BasicNameValuePair("response_type", "code"));
+        params.add(new BasicNameValuePair("scope", "openid"));
+        params.add(new BasicNameValuePair("state", "state-ipv"));
+        response =
+                IpvCoreStubUtil.sendAuthorizationRequest(DEV_AUTHORIZATION_URI, sessionId, params);
+    }
+
+    @And("a valid authorization code is returned in the response")
+    public void aValidAuthorizationCodeIsReturnedInTheResponse() throws IOException {
+        JsonNode jsonNode = objectMapper.readTree(response.body());
+        assertNotNull(jsonNode.get("authorizationCode").get("value").textValue());
+        assertNotNull(jsonNode.get("redirectionURI").textValue());
+        assertNotNull(jsonNode.get("state").get("value").textValue());
+    }
+
+    @When("user sends a request to authorization end point with invalid client id")
+    public void user_sends_a_request_to_authorization_end_point_with_invalid_client_id()
+            throws URISyntaxException, IOException, InterruptedException {
+        List<NameValuePair> params = new ArrayList<>();
+        params.add(
+                new BasicNameValuePair(
+                        "redirect_uri",
+                        "https://di-ipv-core-stub.london.cloudapps.digital/callback"));
+        params.add(new BasicNameValuePair("client_id", "INVALID-CLIENT-ID"));
+        params.add(new BasicNameValuePair("response_type", "code"));
+        params.add(new BasicNameValuePair("scope", "openid"));
+        params.add(new BasicNameValuePair("state", "state-ipv"));
+        response =
+                IpvCoreStubUtil.sendAuthorizationRequest(DEV_AUTHORIZATION_URI, sessionId, params);
+    }
+
+    @And("a validation message is sent in the response")
+    public void aValidationMessageIsSentInTheResponse() throws IOException {
+        JsonNode jsonNode = objectMapper.readTree(response.body());
+        assertEquals(1019, jsonNode.get("code").asInt());
+        assertEquals("Session Validation Exception", jsonNode.get("message").asText());
+        assertEquals("1019: Session Validation Exception", jsonNode.get("errorSummary").asText());
+    }
+
+    @When("user sends a request to authorization end point with invalid redirect uri")
+    public void userSendsARequestToAuthorizationEndPointWithInvalidRedirectUri()
+            throws URISyntaxException, IOException, InterruptedException {
+        List<NameValuePair> params = new ArrayList<>();
```

Comment:
Remove lines 127-132
&
just pass in the redirectUri and clientId to the `sendAuthorizationRequest` method. See comment on `IpvCoreStubUtil` class